src/core/ui.js

```diff
@@ -24,6 +24,14 @@ styleElement.classList.add("removeOnSave");
 
 document.head.appendChild(styleElement);
 
+let body = $('body');
+let fileIssueLink = $('<a href="https://github.com/w3c/respec/issues/new?title=&body=" id="fileIssueLink">file an issue related to the text selected</a>');
+body.append(fileIssueLink);
+body.mouseup(function () {
+  let textSelected = window.getSelection().toString();
+  fileIssueLink.attr('href',`https://github.com/w3c/respec/issues/new?title='${textSelected}'&body=${textSelected}`)
```

Comment:
Naturally, you want the spec's "issueBase" instead of ReSpec issue's URL 😉

Reply:
Please construct the URL using "new URL()" and then set the `title` and `body` via the URL's `searchParams` property. Then assign the resulting string via `a.href = fileIssueLink.href;`

Reply:
OK will do this too.

Reply:
How do I get the `issueBase` of the `spec` ?